### PR TITLE
Update readme description for Kodi 19 and add table of contents

### PR DESCRIPTION
**Pull request by phunkyfish**
v7.3.3
- Update: Readme - update description for Kodi 19 and add table of contents

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,23 +19,34 @@ This addon leverages the OpenWebIf project to interact with the Enigma2 device v
...
+* `1.3.5`: Embedded EPG Genres, Tuner Details, Provider Name, Picon URLs
+* `1.3.6`: Editing Recordings
+* `1.3.7`: Backend Channel Numbers
+
+The contents of this README.md file are as follows:
+
+1. [Build instructions](#build-instructions)
+2. [Support links](#support-links)
+3. [Compatibility with OpenWebIf and other WebIfs](#compatibility-with-openwebif-and-other-webifs)
+3. [Configuring the addon](#configuring-the-addon)
```
Comment (AlwinEsch) [inline README.md]: Here you have a small typo, two times 3 there (as note markdown itself correct it and shows 4).

File: README.md [hunk 2]
```diff
@@ -5,14 +5,9 @@
...
 
-## Overview
-Enigma2 is a open source TV-receiver/DVR platform which Linux-based firmware (OS images) can be loaded onto many Linux-based set-top boxes (satellite, terrestrial, cable or a combination of these) from different manufacturers.
-
-This addon leverages the OpenWebIf project to interact with the Enigma2 device via Restful APIs: (https://github.com/E2OpenPlugins/e2openplugin-OpenWebif)
-
-### Compatibility
+Supports streaming of Live TV & Recordings with features such as Timeshifting, EPG, Timers and Autotimers. In addition the majority of Enigma2 devices support viewing and recording IPTV streams. The add-on supports IPTV by simply passing the URL to [inputstream.ffmpegdirect](#https://github.com/xbmc/inputstream.ffmpegdirect), allowing a local timeshift feature similar to native Enigma2 streams.
```
Comment (AlwinEsch) [inline README.md:10]: Maybe you can add line breaks here and other places to prevent this very long line.
Markdown correct them and shows as one.

**Review by AlwinEsch — COMMENTED** [review]
Wow, thats a really good addon documentation, Cool :sunglasses:

File: README.md (hunk 1 above)
Comment (phunkyfish) [reply README.md]: Nice catch, thanks!

File: README.md (hunk 2 above)
Comment (phunkyfish) [reply README.md:10]: When I added these for iptvsimple it wrapped correctly so I think it should be ok here too. Sure, I'll merge and if there is still a problem I can add a follow up commit.

Comment (phunkyfish) [reply README.md:10]: Do you have a screenshot of what it looks like for you?

Comment (AlwinEsch) [reply README.md:10]: It relates to the editor/textviewer app from where it becomes looked, normally a good editor makes always itself breaks on end of view.
Only not all editors have it. Like on code the clang line length reason.

As example by editor of midnight commander:
Bildschirmfoto vom 2021-02-20 14-49-29
Bildschirmfoto vom 2021-02-20 14-50-29

Comment (phunkyfish) [reply README.md:10]: Ah, I understand now. You mean in an IDE. I had assumed it was when viewing on GitHub!

**Pull request merged**